graph_nets/tests_tf2/blocks_test.py

```diff
@@ -405,6 +406,30 @@ def test_output_values(
     self.assertNDArrayNear(
         expected_output_edges.numpy(), output_graph_out.edges.numpy(), err=1e-4)
 
+  @parameterized.named_parameters(
+      ("only scaling", 2, 0),
+      ("only offsetting", 0, 2),
+      ("scaling and offsetting", 2, 2),
+      ("without scaling and offsetting", 1, 0),
+  )
+  def test_optional_arguments(self, scale, offset):
+    """Compares the output of an EdgeBlock to an explicit computation based on optional arguments."""
```

Comment:
Our code formatting requires keeping every line <= 80 characters.

Could you:
* Summarize one line docstrings to keep them sorter.

* Indent full docstrings (those with Args and Returns) into multiple lines similarly to the existing docstrings.

* split multiple long code lines into multiple lines right after the commas brackets, etc. avoiding "\" if possible, following the other examples in the file.